### PR TITLE
Added Universal wireless communication library

### PR DESCRIPTION
**Pull request by siravijbb**
**Always tag `🚀 READY FOR REVIEW` once your PR is Ready to be one of us :)**

We value your endeavors to enhance the Hacktoberfest Museum better!

## If (this.PR.category === "The new entry 💫")

**Please tag `The new entry 💫` to the PR in order to categorize yours as the star being recognized by the Museum.**

## Else if (this.PR.category === "Enhancement 🛠")

**Please tag `Enhancement 🛠` to the PR as you're developing the Museum more wonderfully!**

## Finally:

> 🪄 **DON'T FORGET TO CLAIM YOUR TICKET!** via [Eventpop application form.](https://www.eventpop.me/events/13854/application_forms/845/applicants/new)

### CODE REVIEW

**Review by Gusb3ll — CHANGES_REQUESTED** [review]
Did you do any [contribution](https://github.com/jgromes/RadioLib/graphs/contributors) to the project or has any relation to it?, If you wanted to continue then please format your code using `pnpm prettier:write` kub

**Comment by Gusb3ll** [comment]
I think I've said before to check for formatting before pushing, please do it on your next PR kub

**Comment by siravijbb** [comment]
> Did you do any [contribution](https://github.com/jgromes/RadioLib/graphs/contributors) to the project or has any relation to it?, If you wanted to continue then please format your code using `pnpm prettier:write` kub

eh,I open issue for them,But why `pnpm prettier:write` it said 'The filename, directory name, or volume label syntax is incorrect.'

**Comment by siravijbb** [comment]
image

**Comment by Gusb3ll** [comment]
🤔

**Comment by leomotors** [comment]
My suggestion

https://learn.microsoft.com/en-us/windows/wsl/install

or

https://www.apple.com/th/mac/

File: src/entries/Universal-wireless-communication-library.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  title: 
+    "Universal wireless communication library",
+  theme: ProjectThemes.Mindful,
+  founderName: Founders.jgromes,
+  topic: AcademicTopics.Arduino,
+  githubUrl: "https://github.com/jgromes/RadioLib",
+  imagePath: "/images/jgromes.png",
+};
+
+export default props;
```
Comment (leomotors) [inline src/entries/Universal-wireless-communication-library.ts]: No CRLF

**Review by leomotors — CHANGES_REQUESTED** [review]

**Pull request closed without merging**